iree/compiler/Dialect/HAL/IR/test/interface_ops.mlir

```diff
@@ -0,0 +1,46 @@
+// RUN: iree-opt -allow-unregistered-dialect -split-input-file %s | iree-opt -allow-unregistered-dialect -split-input-file | IreeFileCheck %s
+
+// CHECK-LABEL: @interface_workgroup_info
+func @interface_workgroup_info() {
+  // CHECK: %workgroup_id_x = hal.interface.workgroup.id[0] : index
+  %0 = hal.interface.workgroup.id[0] : index
+  // CHECK: %workgroup_count_y = hal.interface.workgroup.count[1] : index
+  %1 = hal.interface.workgroup.count[1] : index
+  // CHECK: %workgroup_size_z = hal.interface.workgroup.size[2] : index
+  %2 = hal.interface.workgroup.size[2] : index
+  return
+}
+
+// -----
+
+// CHECK-LABEL: @interface_io_tensors
+func @interface_io_tensors() {
```

Comment:
Nit: Wanna move these tests to a different commit in your stack ?

Reply:
ah this is just moving them while touching the file - git absorb issues probably :)